src/validateForm.js

```diff
@@ -1,5 +1,5 @@
 const validateForm = (formData, nextPage) => {
-  if (!formData) {
+  if (formData.length == 0) {
```

Comment:
Best condition is `(!formData || !formData.length)`

Explaination: false = 0, true = anything else, so you can just check if formData.length === false (0) => !formData.length